### PR TITLE
Rustfmt-ing liblog v2.

### PR DESCRIPTION
**Pull request by goyox86**
Hi Rustaceans!

This is the second version of running rustfmt on liblog https://github.com/rust-lang/rust/pull/28898. There are only rustfmt suggested changes. Because I think that some patterns here (that I fixed up in the prev PR need to be discussed in detail. 

//cc @nrc

### CODE REVIEW

File: src/liblog/lib.rs [hunk 1]
```diff
@@ -463,49 +460,72 @@ mod tests {
...
     }
 
     #[test]
     fn zero_level() {
-        let dirs = [
-            LogDirective { name: None, level: 3 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 0 }
-        ];
+        let dirs = [LogDirective {
+            name: None,
```
Comment (cristicbz) [inline src/liblog/lib.rs]: This seems wrong, it should be indented relative to beginning of `LogDirective`.

File: src/liblog/lib.rs [hunk 2]
```diff
@@ -445,16 +444,14 @@ mod tests {
...
-            LogDirective {
-                name: Some("crate1::mod1".to_string()),
-                level: 2
-            }
-        ];
+        let dirs = [LogDirective {
+            name: Some("crate2".to_string()),
+            level: 3,
+        },
+                    LogDirective {
```
Comment (alexcrichton) [inline src/liblog/lib.rs]: Hm it looks like formatting may have gone a little haywire here?

File: src/liblog/lib.rs [hunk 3]
```diff
@@ -301,11 +303,10 @@ pub fn log(level: u32, loc: &'static LogLocation, args: fmt::Arguments) {
...
     // set during logging.
-    let mut logger: Box<Logger + Send> = LOCAL_LOGGER.with(|s| {
-        s.borrow_mut().take()
-    }).unwrap_or_else(|| {
-        box DefaultLogger { handle: io::stderr() }
-    });
+    let mut logger: Box<Logger + Send> = LOCAL_LOGGER.with(|s| s.borrow_mut().take())
+                                                     .unwrap_or_else(|| {
+                                                         box DefaultLogger { handle: io::stderr() }
+                                                     });
```
Comment (alexcrichton) [inline src/liblog/lib.rs]: Perhaps this could stay as is for now?

File: src/liblog/directive.rs [hunk 4]
```diff
@@ -40,44 +42,48 @@ pub fn parse_logging_spec(spec: &str) -> (Vec<LogDirective>, Option<String>) {
...
+            if s.is_empty() {
+                continue
+            }
+            let mut parts = s.split('=');
+            let (log_level, name) = match (parts.next(),
+                                           parts.next().map(|s| s.trim()),
+                                           parts.next()) {
+                (Some(part0), None, None) => {
                 // if the single argument is a log-level string or number,
                 // treat that as a global fallback
```
Comment (nrc) [inline src/liblog/directive.rs]: These comments are misaligned and need fixing up manually (there is a fix for rustfmt in the works for this).

File: src/liblog/lib.rs (hunk 3 above)
Comment (nrc) [inline src/liblog/lib.rs]: This looks OK to me. I don't think it is better than the original, but I wouldn't say its worse. And I'm starting to appreciate having the function names aligned like this.

File: src/liblog/lib.rs (hunk 2 above)
Comment (nrc) [inline src/liblog/lib.rs]: Yeah, this is broken. But I think a fix has already landed in Rustfmt. @goyox86, could you try re-running and see if this gets fixed?

https://github.com/nrc/rustfmt/pull/449

**Comment by nrc** [comment]
I'd like to see if https://github.com/nrc/rustfmt/pull/449 fixes the odd formatting in the tests. The only other thing is the comment which needs fixing up, otherwise looks good!

**Comment by goyox86** [comment]
@nrc, @alexcrichton I've run latest rustfmt and semmed to fix the issues. Also manually aligned comments.

File: src/liblog/lib.rs (hunk 3 above)
Comment (nikomatsakis) [inline src/liblog/lib.rs]: if I were writing this by hand, I think i'd be inclined to insert a newline after the `=` (though I guess this probably technically fits with whatever rustfmt thinks our max line length is?)

**Comment by nikomatsakis** [comment]
I'm inclined to r+, but I'll wait on what @alexcrichton thinks.

**Comment by nrc** [comment]
LGTM

@goyox86 thanks for the changes!

**Comment by goyox86** [comment]
Sweet! @nrc The CI bot will merge it. Or do I have to merge?

**Comment by alexcrichton** [comment]
@bors: r+ 94946db60d84d1a6a5d45dc7be7e778882637367

Ah no worries @goyox86, now that @bors has the r+ it'll come along eventually and auto-merge

**Pull request merged**